microcosm_postgres/store.py

```diff
@@ -167,6 +164,12 @@ def search(self, *criterion, **kwargs):
         query = self._filter(query, **kwargs)
         return query.all()
 
+    def expunge(self, instance):
+        return self.session.expunge(instance)
+
+    def merge(self, instance, new_instance):
```

Comment:
In the case I care about, we need to manually populate the new instance with an (immutable) value from the old instance that it doesn't have.